include/render/gles2.h

```diff
@@ -37,6 +37,8 @@ struct wlr_gles2_tex_shader {
 	GLint invert_y;
 	GLint tex;
 	GLint alpha;
+	GLint pos_loc;
```

Comment:
Nit: the existing members are locations too, so it's inconsistent to add a `_loc` suffix to the new members, we should drop it

Reply:
It's an attribute location compared to other that are uniform locations. `pos_attrib` maybe?

Reply:
Works for me

Reply:
Done